packages/hooks/src/useOauth2.ts

```diff
@@ -39,9 +44,61 @@ const useOauth2 = ({ handleLogout }: { handleLogout: () => Promise<void> }) => {
     };
 
     const logoutHandler = async () => {
+        if (process.env.REMOTE_CONFIG_URL) {
+            const flags = await fetch(process.env.REMOTE_CONFIG_URL)
+                .then(res => res.json())
+                .catch(() => FIREBASE_INIT_DATA);
```

Comment:
why do we need this whole initialization again?

Reply:
The attributes are not getting updated without initialization.we can check if there is a way without doing the initialization again.